### ISSUE TITLE
iOS Generated PDF isues

### ISSUE THREAD

**Issue by drcoxpediant** [eKoopmans/html2pdf.js#548]
I do have this configuration:
```javascript
 "html2pdf.js": "^0.10.1",

const opt = {
    margin: 0,
    filename: `report_${moment().format('MM_DD_YYYY')}`,
    image: {
      type: 'jpeg',
      quality: 0.98,
    },
    html2canvas: {
      scrollY: 0,
      scrollX: 0,
      dpi: 192,
      letterRendering: true,
      scale: 2,
      logging: true, // while testing
      allowTaint:true,
      useCORS: true,
      removeContainer: false, // while testing
    },
    jsPDF: {
      unit: 'in',
      format: 'letter',
      orientation: 'portrait',
    },
    pagebreak: {
      before: '.pdf-page-break-comp__before',
      after: '.pdf-page-break-comp__after',
      avoid: '.pdf-page-break-comp__avoid',
    },
  };

html2pdf()
    .set(opt)
    .from(containerToBeRendered)
    .toPdf()
    .get('pdf')
    .then((_) => {
      parent.postMessage(
        `pdfReportExecStatus: completed: #downloadedpdf`,
        '*'
      );
    })
    .save();
```

And everything works well on desktop and android devices... The PDF is created as I do expect. But on iOS the PDF bold text looks weird and is adding an extra space to my googleChart and its not rendering correctly the PDF as desktop or android.
**Expected PDF**
![2022Sunday10-53-59](https://user-images.githubusercontent.com/79216700/167307637-4a2f447d-d86b-4607-9833-2ffd00aa84d9.png)

**IOS** 
![2022Sunday11-19-04](https://user-images.githubusercontent.com/79216700/167307801-e32025aa-8378-4eb8-9b03-a49439592d8c.png)

**ANOTHER ISSUE I NOTICED:**
For some reason if I run the PDF creation having the browser dev tools the PDF renders weird. just FYI.

**Comment by drcoxpediant** [comment]
I have found a fix for this.. .using verion ^0.9.3",
